README.rst

```diff
@@ -220,3 +220,16 @@ Contributing
 ------------
 
 Please see our `contributing guide <https://github.com/mne-tools/mne-bids/blob/master/CONTRIBUTING.md>`_.
+
+MEG, EEG, iEEG Data From Non-BIDS Acquisition Systems
+--------------------------------------------------------
```

Comment:
Please reduce the no. of dashed to match the no. of characters in the line above